clients/js/package.json

```diff
@@ -37,6 +37,7 @@
     "build:main": "tsc -p tsconfig.json",
     "build:module": "tsc -p tsconfig.module.json",
     "genapi": "./genapi.sh",
-    "prettier": "prettier --write ."
+    "prettier": "prettier --write .",
+    "release": "run-s build test:run && npm publish"
```

Comment:
shell noob question - if test rails will npm publish not run?

Reply:
yes it will error out. Here is a forced fail to demonstrate.

```shell
yarn run v1.22.19
$ run-s build test:run && npm publish
$ run-s clean build:*
$ rimraf dist
$ tsc -p tsconfig.json
$ tsc -p tsconfig.module.json
$ jest --runInBand
 FAIL  test/client.test.ts
  ● it should get the heartbeat

    expect(received).toBe(expected) // Object.is equality

    Expected: false
    Received: true

      18 |     expect(heartbeat).toBeDefined();
      19 |     expect(heartbeat).toBeGreaterThan(0);
    > 20 |     expect(true).toBe(false);
         |                  ^
      21 | });
      22 |
      23 | test("it should reset the database", async () => {

      at Object.<anonymous> (test/client.test.ts:20:18)

 PASS  test/add.collections.test.ts
 PASS  test/get.collection.test.ts
 PASS  test/collection.client.test.ts
 PASS  test/query.collection.test.ts
 PASS  test/collection.test.ts
 PASS  test/upsert.collections.test.ts
 PASS  test/delete.collection.test.ts
 PASS  test/update.collection.test.ts
 PASS  test/peek.collection.test.ts

Test Suites: 1 failed, 9 passed, 10 total
Tests:       1 failed, 34 passed, 35 total
Snapshots:   0 total
Time:        2.078 s
Ran all test suites.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
ERROR: "test:run" exited with 1.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```